Tests/LibWeb/Text/input/DOM/range-get-client-rects.html

```diff
@@ -0,0 +1,21 @@
+<!--refer to https://wpt.live/css/cssom-view/DOMRectList.html-->
+<body>
+<div id="x">x</div>
+<script src="../include.js"></script>
+<script>
+    function print_class_string(object) {
+        const str = {}.toString.call(object);
+        const res = str.substring(8, str.length - 1);
+        println(res)
+    }
```

Comment:
it took me a moment to realise `str.substring(8, str.length - 1)` is needed to extract a name. maybe let's simply print `[object <name>]`?